internal/controllers/machine/machine_controller.go

```diff
@@ -525,7 +525,8 @@ func (r *Reconciler) drainNode(ctx context.Context, cluster *clusterv1.Cluster,
 			log.Error(err, "Could not find node from noderef, it may have already been deleted")
 			return ctrl.Result{}, nil
 		}
-		return ctrl.Result{}, errors.Errorf("unable to get node %q: %v", nodeName, err)
+		return ctrl.Result{}, errors.Wrapf(err, "unable to get node %q: %v", nodeName )
```

Comment:
```suggestion
		return ctrl.Result{}, errors.Wrapf(err, "unable to get node %q", nodeName )
```

My fault this time 🙂. The suggestion was off.